themes/base/accordion.css

```diff
@@ -21,3 +21,14 @@
 	border-top: 0;
 	overflow: auto;
 }
+
+/* RTL support */
+.ui-accordion-rtl {
+	direction: rtl;
```

Comment:
Why does this need to exist? The direction is already set or we wouldn't be adding this class.